packages/plugin-pnpm/sources/PnpmLinker.ts

```diff
@@ -229,19 +235,18 @@ class PnpmInstaller implements Installer {
 
           const storeEntryPath = ppath.join(storeLocation, storeEntry as Filename);
 
-          removals.push(xfs.readdirPromise(storeEntryPath)
-            .then(entries => {
-              return Promise.all(entries.map(async entry => {
-                const p = ppath.join(storeEntryPath, entry);
-                if (entry === Filename.nodeModules) {
-                  const extraneous = await getNodeModulesListing(p);
-                  extraneous.delete(identComponents.join(ppath.sep) as PortablePath);
-                  return cleanNodeModules(p, extraneous);
-                } else {
-                  return xfs.removePromise(p);
-                }
-              }));
-            })
+          removals.push(xfs.readdirPromise(storeEntryPath).then(entries => {
```

Comment:
Shouldn't the `.catch` at least be directly next to the closing paren if the `.then` got moved to the same line as the async function call?

Reply:
Yes, I don't know why this code changed, I didn't do it 🤔